### ISSUE TITLE
Harvesting Person Records Associated with a Locality

### ISSUE THREAD

**Issue by PatientRowan** [historyforge/historyforge#317]
@EveSnyder were playing with this yesterday ‘cause it seems like a really great feature.  Eve wanted to harvest a list of all person records associated with Enfield and park it somewhere.  So we enable Localities in the Fields list:

![IMG_0600](https://github.com/historyforge/historyforge/assets/114665933/8e750848-e10e-4669-bd76-d2cca43e9eb1)

And then we want to search by Filter on Enfield.  But Locality is not an option in the drop-down list.

So OK, maybe we can select Enfield this way?

![IMG_0601](https://github.com/historyforge/historyforge/assets/114665933/bc87c499-3cb5-4a15-b9be-f76275d279b0)

This does seem to result in a list of Enfield-related person records, but the total count is 80,000 that is the complete person record database.  You can scroll to the end of the fairly small Enfield-related person record  list; it might be in the 100s. 
Eve, ever hopeful, exported the supposed Enfield list to a CSV file.  Her computer said it was loading and loading and it never loaded, probably because it was working on all 80,000.

**Comment by dfurber** [comment]
It's interesting because the list shown on the screen only has the Enfield peeps.

**Comment by dfurber** [comment]
Harvesting people, just doesn't sound right.

**Comment by EveSnyder** [comment]
“Soylent green is people!”

Eve Snyder, PhD
Historian and Project Director, HistoryForge
<https://tompkins.historyforge.net/>
*The History Center in Tompkins County  <http://thehistorycenter.net/>*
110 North Tioga Street
in the Tompkins Center for History & Culture
Ithaca, NY 14850

Follow HistoryForge on social media:
Facebook: *https://www.facebook.com/ExploreHistoryForge/
<https://www.facebook.com/ExploreHistoryForge/>*
Twitter: *https://twitter.com/_HistoryForge
<https://twitter.com/_HistoryForge>*
LinkedIn: https://www.linkedin.com/showcase/historyforge


<https://www.facebook.com/tompkinshistory>
<https://twitter.com/TompkinsHistory>
<https://www.instagram.com/tompkinshistory/>
<https://www.youtube.com/channel/UCO-0oMlJzAm0Xb8zYx5az4w>
<https://www.linkedin.com/company/tompkins-history>   @tompkinshistory

**Comment by PatientRowan** [comment]
The Localities limit is fantastic, it’s only a few days and I already am using it.  Example: I am writing a short intro to person records for transcribers in Caroline, and I want to show them that when they write interesting notes about their persons on the census record, I might very well delete the notes from the census record and transfer them to the person record.  One of the Caroline transcribers had pointed out that her person had a great photo in Find a Grave.  I was experimenting with uploading photos at the time, and I uploaded the photo to HistoryForge as a test.  (This is NOT something we would do wholesale and I am fine with deleting the photo as need be—but she is a fine-looking woman, pioneering women’s pants!). 
Having been caught out forgetting the name of the person, I tried to pull out the photo with a fancy search.  There’s only one photo associated with Caroline, but it never loads for me:
![IMG_0614](https://github.com/historyforge/historyforge/assets/114665933/e3d4a0a4-de5c-4e14-9104-e46a61e2c4f6)
